### PR TITLE
Kids First DRS Signed URLs [BT-172] [BT-218]

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

File: martha/martha_v3.js
Comment (kshakir) [inline martha/martha_v3.js]: This should be something like (comments need re-english)
```javascript
bondProvider && // Only retrieve the SA if the DRS provider uses a Bond provider for auth
[ACCESS_METHOD_TYPE_NONE, ACCESS_METHOD_TYPE_GCS].includes(accessMethodType) && // Only retrieve the SA for projects that implicitly or explicitly use GCS for accessing the data
```

File: test/martha/martha_v3.test.js
Comment (kshakir) [inline test/martha/martha_v3.test.js]: While making other edits, please add a newline between checks for `getCall(0)` and `getCall(1)` to match the spacing between `getCall(2)` below.

File: test/martha/martha_v3.test.js
Comment (kshakir) [inline test/martha/martha_v3.test.js]: Minor whitespace consistency again: add a space before and after the checks during `getCall(1)` to separate the section from `getCall(0)` and `getCall(2)`.

**Review by breilly2 — COMMENTED** [review]
Looks good to me so far. The only question I know of is if our suggestion of `kids-first` as the name for the new bond provider will be used. Hopefully we can get a definitive answer to that before we find ourselves blocked by not merging this PR.

**Review by kshakir — APPROVED** [review]

**Comment by mcovarr** [comment]
jenkins retest

**Review by kshakir — APPROVED** [review]
Re-👍

**Review by breilly2 — APPROVED** [review]
👍

**Pull request merged**
